### PR TITLE
[Easy] Allow withdraw scripts to be directly executed onchain

### PR DESCRIPTION
**Pull request by fleupold**
Similar to #519 but for withdrawals. This will allow us to easily relaunch strategies until the Safe Web is available.

### Test Plan

Withdraw my test liquidity (between WETH and wxDAI)

```sh
yarn truffle exec scripts/request_withdraw.js --masterSafe=$MASTER_SAFE --brackets=0x5D6b329918c2BF4d9B722c5c8F8d981E61974d03,0x52033e1F448C69eD6AdDCebF47f9a5836df6395D,0x9A5aB9da6Ee6D9907B19738824C25dfAf0482CD6,0x876EF54A5833DaF55F2c27b4fD098cfdbdb92b70,0x0BdB46E41c80749F5506f51037a981276Dd982E4,0x463Ba08241022F65416a57b428EE32cB25402918,0xf1E115BB04789f85A85f033be932953Cb09DFc72,0xAB795A52D809e4Aa56da23b215Db7663d57CDE3A,0x45d8a3d10cd7Da86Dac21a51B305312Ebf80072d,0x2952bC69739c9534d09525ccCa33950F311CbE88 --tokenIds=1,4 --network=$NETWORK_NAME --executeOnchain --noBalanceCheck
# Wait 5 minutes
yarn truffle exec scripts/claim_withdraw.js --masterSafe=$MASTER_SAFE --brackets=0x5D6b329918c2BF4d9B722c5c8F8d981E61974d03,0x52033e1F448C69eD6AdDCebF47f9a5836df6395D,0x9A5aB9da6Ee6D9907B19738824C25dfAf0482CD6,0x876EF54A5833DaF55F2c27b4fD098cfdbdb92b70,0x0BdB46E41c80749F5506f51037a981276Dd982E4,0x463Ba08241022F65416a57b428EE32cB25402918,0xf1E115BB04789f85A85f033be932953Cb09DFc72,0xAB795A52D809e4Aa56da23b215Db7663d57CDE3A,0x45d8a3d10cd7Da86Dac21a51B305312Ebf80072d,0x2952bC69739c9534d09525ccCa33950F311CbE88 --tokenIds=1,4 --network=$NETWORK_NAME --executeOnchain
```

### CODE REVIEW

**Review by bh2smith — APPROVED** [review]
Really nice clean and easy change! Looks good.

**Comment by bh2smith** [comment]
NIT: I think the first command in your test plan should be `request_withdraw`

File: scripts/request_withdraw.js [hunk 1]
```diff
@@ -17,7 +18,8 @@ module.exports = async (callback) => {
     const transaction = await prepareWithdrawRequest(argv, true)
     const answer = await promptUser("Are you sure you want to send this transaction to the EVM? [yN] ")
     if (answer == "y" || answer.toLowerCase() == "yes") {
-      await signAndSend(await masterSafe, transaction, argv.network)
+      const signAndSendOrExecuteOnChain = argv.executeOnchain ? async (safe, tx) => signAndExecute(safe, tx) : signAndSend
```
Comment (fedgiac) [inline scripts/request_withdraw.js:21]: Same as before:
```suggestion
      const signAndSendOrExecuteOnChain = argv.executeOnchain ? (safe, tx) => signAndExecute(safe, tx) : signAndSend
```

File: scripts/wrapper/withdraw.js [hunk 2]
```diff
@@ -299,6 +299,11 @@ module.exports = function (web3, artifacts) {
         return str.split(",")
       },
     })
+    .option("executeOnchain", {
+      type: "boolean",
+      default: false,
+      describe: "Directly execute transaction on-chain instead of sending to the backend",
+    })
```
Comment (fedgiac) [inline scripts/wrapper/withdraw.js:306]: This `argv` is also used in `transfer_funds_to_master.js`, can you also modify that file to make it consistent with the other withdrawing scripts?

**Review by fedgiac — APPROVED** [review]
Looks good!

File: scripts/wrapper/withdraw.js (hunk 2 above)
Comment (fleupold) [reply scripts/wrapper/withdraw.js:306]: I'm not sure what you mean by this? All withdraw related scripts import the argvs from this file, so it should already be available for the transfer script.

Comment (fedgiac) [reply scripts/wrapper/withdraw.js:306]: Sorry, I missed a file, all good!

**Pull request merged**